.travis.yml

```diff
@@ -106,8 +110,31 @@ script:
  - echo; echo "${green}EVERYTHING SEEMED TO PASS!${normal}"
 matrix:
   fast_finish: true
-  # Rule for running Coverity Scan.
   include:
+    # Test with GCC7
+    - os: linux
+      addons:
+        apt:
+          sources:
+            - ubuntu-toolchain-r-test
+          packages:
+            - *common_deps
+            - g++-7
+      env:
+        - MATRIX_EVAL='CC=gcc-7 && CXX=g++-7'
```

Comment:
I think you can use global and matrix here to get rid of the TEST_GROUP=1 rule.

Something like that (not tested):

env:
  global:
    - MATRIX_EVAL='CC=gcc-7 && CXX=g++-7'
    - TRAVIS_CVC4=yes TRAVIS_CVC4_CHECK_PORTFOLIO=yes TRAVIS_WITH_LFSC=yes TRAVIS_CVC4_JAVA_API_TEST=yes TRAVIS_CVC4_CONFIG='debug --enable-language-bindings=java,c --with-lfsc --with-portfolio'
  matrix:
    - TEST_GROUP=0
    - TEST_GROUP=1